FeedAPIChallenge/Feed Feature/FeedImagesMapper.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  FeedImagesMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by GD on 8/24/21.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+final class FeedImagesMapper {
+	private struct Root: Decodable {
+		let items: [Item]
+
+		var feedImages: [FeedImage] {
+			return items.map { $0.item }
+		}
+	}
+
+	struct Item: Decodable {
```

Comment:
`Item` can be declared as private since it's been used only by `Root` which is also a private struct.

```suggestion
	private struct Item: Decodable {
```